Symfony/CS/Error/Internal.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Error;
+
+class Internal extends AbstractError
```

Comment:
One can distinguish it via some flag.
When we need to see the difference between internal and external we create separated class. But we create only a flag for linting error and exception error.
Why such a inconsistency?